LICENSE

```diff
@@ -0,0 +1,231 @@
+COPYING -- Describes the terms under which DeXter-on-Radix is distributed.
+A copy of the GNU GPL is appended to this file.
+
+IMPORTANT DeXter-on-Radix LICENSE TERMS
+
```

Comment:
Do we really need these 5 lines, IMO we could remove those as they do not add anything that's already covered in the license.

Reply:
As you prefer, I added it as a header, because I saw other GPL-licensed projects doing so

Reply:
We should remove all lines up to 11 and keep only the official license text. The previous content is redundant and repeats what's in the license (please correct me.if I'm wrong). Other projects often use a LICENSE file for the official text and a separate COPYING file for additional requirements. 

I'm confused why we include COPYING information inside the LICENSE file and why we have it at all without extra requirements. It's clearer to have just the license text so developers know exactly which license applies with no additional conditions.

Reply:
That being said, I am open to keep lines 6-11, as they serve as a nice summary of the license itself. So people that do not know about licenses can get a quick overview.